### DISCUSSION TITLE
Wizard Looping

### DISCUSSION THREAD

**Discussion by sh-dian** [filamentphp/filament#11546] [Help]
### Package

Form builder

### Package Version

v3

### How can we help you?

What im trying to do here is im using wizard for step, then at one step, i need to select multiple notation, so here after i loop all the selected notation, i need to select. the problem now the data come out null for the select part. anyone can help me?
![image](https://github.com/filamentphp/filament/assets/90387094/7967b730-e361-400b-81e2-a1546f0cb9a7)
![image](https://github.com/filamentphp/filament/assets/90387094/4fbb6201-ebd1-48aa-b770-4e1314f6f9c9)

**Comment by leandrocfe** [comment]
In my opinion, you should use 'hazard_0' instead of 'hazard 0' as id.
Are you using the panel builder or the form builder only? Share the whole code you are trying to do (code, not image)
